README.md

```diff
@@ -1,46 +1,23 @@
-## Seva Kitchen Website Project
-
-
-### Introduction
-
-Check out the introduction video to know why we are doing what we are doing and how it will impact Seva Kitchen.
-
-[![Introduction to the Seva Kitchen website project](https://i.imgur.com/01NCCuS.png)](https://www.youtube.com/watch?v=ub3wKcvwhUo&feature=youtu.be "Introduction to the Seva Kitchen project")
-
-
-### Contribution Instructions
-
-- Create a branch from the master branch with the following format - (your name) + "/" (few words for task you are working on). For eg. "saumitra/initiative-card"
-
-- Once you are done with your task, push your changes in your branch.
-
-- Raise a pull request from your branch to master.
-
-- Your code changes will be reviewed by mentors. Please push the changes as suggested by mentors, unless you disagree, in which case, we encourage you to make your case.
-
-- If you saw a lot of foreign words in these instructions, don't worry, there are many awesome GitHub guides on the internet that cover all these buzz words. One of them is here - http://rogerdudler.github.io/git-guide/ . This will get you started right away.
-
-
```

Comment:
Can we not remove this?

Reply:
I think this readme.md is common for all of us...

Reply:
yes. and in your PR, this is getting removed. I feel that this is an important text which should not be removed.